setup.py

```diff
@@ -3,7 +3,7 @@
 
 setup(
     name="reasoner-transpiler",
-    version="1.11.11",
+    version="2.0.0",
     author="Patrick Wang",
     author_email="patrick@covar.com",
```

Comment:
Let's change the author to  somebody that can still be contacted

Reply:
Hi Chris, I have added (https://github.com/ranking-agent/reasoner-transpiler/blob/trapi-1.4/setup.py#L9-L10),  based on this (https://docs.python.org/3/distutils/setupscript.html#additional-meta-data)

Reply:
i.e maintainer and maintainer_email feilds